ci/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:buster-slim
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+
+# hadolint ignore=DL3008
+RUN apt-get update --fix-missing && \
+    apt-get install -y --no-install-recommends wget build-essential bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion sudo && \
```

Comment:
Those dependencies, are they all needed for the CI image? E.g. hg and svn seem to be pretty random. What's libxext6, libsm6 and libxrender1 for?

Reply:
```suggestion
    apt-get install -y --no-install-recommends tini wget build-essential bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion sudo && \
```

Reply:
the dependencies are replicated from the miniconda image ... I assume that some tests install conda packages that depend on those... the libs are the client libs regarding using X11 functionality used by conda packages.